tests/plugin_tests/chowdsp_plugin_utils_test/GlobalSettingsTest.cpp

```diff
@@ -263,7 +269,11 @@ class GlobalSettingsTest : public TimedUnitTest
                     auto propId = "thread2_" + std::to_string (i);
                     chowdsp::GlobalPluginSettings::SettingProperty prop { propId, i };
                     thread2Settings->addProperties ({ prop });
-                    expectEquals (thread2Settings->getProperty<int> (prop.first), prop.second.get<int>(), "Property is incorrect within thread 2");
+                    juce::MessageManager::callAsync (
+                        [test = thread2Settings->getProperty<int> (prop.first) == prop.second.get<int>()]
+                        {
+                            REQUIRE_MESSAGE (test, "Property is incorrect within thread 2");
+                        });
```

Comment:
So it turns out (I only learned this a couple of minutes ago), that Catch's `REQUIRE` macros can only be run on one thread at a time, although for some reason, this test seemed to be running fine on Mac.

I don't love doing it this way, butI think it might be the best (only?) solution.